packages/ketcher-core/src/application/render/restruct/resgroup.ts

```diff
@@ -528,10 +527,12 @@ function getBracketParamersWithCrossBondsMoreThan2OnOneAtom(
   // if bonds direction is clockwise, then negated
   const needNegated =
     Vec2.crossProduct(bondDirections[0], bondDirections[1]) > 0;
-  crossBondsValues = crossBondsValues.sort((id1, id2) => {
-    notTemplateShapeFirstAtom = Math.abs(Number(id1) - Number(id2)) === 1;
-    return notTemplateShapeFirstAtom && !needNegated ? -1 : 0;
-  });
+  notTemplateShapeFirstAtom =
+    Math.abs(Number(crossBondsValues[0]) - Number(crossBondsValues[1])) === 1;
```

Comment:
I am just not sure if there is case that there are more than 2 elements in crossBondsValues?

Reply:
Sure. Above in the code base we have additional check before calling this function:
`else if (crossBondsValues.length === 2 && crossBondsPerAtom.length === 2) { `

Reply:
So, in any case there are 2 elements in Array

Reply:
Nevertheless, I added typescript tuple [number, number]